### PR TITLE
mod: Capsule temporary improvments

### PR DESCRIPTION
**Pull request by Norr2**
**Temporary improvements while a better solution for phasing and ranged hitting past shields is found**

1. Body Capsule
Resized crouch to same as standing to avoid glitching into spots

2. Enemy Capsule
Increased size to reduce glancing and reduce ranged shooting through shield window
Still smaller than native

3. Friend Capsule
Increase in size closer to enemy capsule

4. Environment/Entity Capsule
Increased slighty, should be less small unintended gaps in maps now

### CODE REVIEW

File: src/Module.Server/ModuleData/native_parameters.xml [hunk 1]
```diff
@@ -14,5 +14,8 @@
     <native_parameter id="running_backward_speed_multiplier" value="0.65" />  <!-- Native = 0.5 -->
     <native_parameter id="running_sides_speed_multiplier" value="0.775" />   <!-- Native = 0.65-->
     <native_parameter id="ladder_climb_up_speed_multiplier"  value="0.88" /> <!-- Native = 0.68-->
+    <native_parameter id="humanoid_physics_capsule_radius_multiplier_for_friend" value="1.3" /> <!--Native = 0.95-->
+    <native_parameter	id="humanoid_physics_capsule_radius_multiplier_for_enemy"	value="1.8" /> <!--Native = 1.5-->
```
Comment (namidaka) [inline src/Module.Server/ModuleData/native_parameters.xml:18]: Do 1.6 for an equivalent end capsule of 0.29.

File: src/Module.Server/ModuleData/native_parameters.xml [hunk 2]
```diff
@@ -14,5 +14,8 @@
     <native_parameter id="running_backward_speed_multiplier" value="0.65" />  <!-- Native = 0.5 -->
     <native_parameter id="running_sides_speed_multiplier" value="0.775" />   <!-- Native = 0.65-->
     <native_parameter id="ladder_climb_up_speed_multiplier"  value="0.88" /> <!-- Native = 0.68-->
+    <native_parameter id="humanoid_physics_capsule_radius_multiplier_for_friend" value="1.3" /> <!--Native = 0.95-->
```
Comment (namidaka) [inline src/Module.Server/ModuleData/native_parameters.xml:17]: You can do 1.5 here as the capsule are tight enough for us to have similar capsule for friend or enemy

**Review by namidaka — CHANGES_REQUESTED** [review]

File: src/Module.Server/ModuleData/native_parameters.xml (hunk 1 above)
Comment (Norr2) [reply src/Module.Server/ModuleData/native_parameters.xml:18]: Thanks for the response, I believe 1.6 is not enough in the short term, with ranged shooting past shields and people getting too close into each other causing more glancing?
But yes 1.6 will help a bit towards this

File: src/Module.Server/ModuleData/native_parameters.xml (hunk 2 above)
Comment (Norr2) [reply src/Module.Server/ModuleData/native_parameters.xml:17]: Ok :)

File: src/Module.Server/ModuleData/native_parameters.xml (hunk 1 above)
Comment (namidaka) [reply src/Module.Server/ModuleData/native_parameters.xml:18]: please start with 1.6

**Pull request merged**